doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
+      * Branches in the nodejs/node repository are only for release lines
     * [See "Updating Node.js from Upstream"](./onboarding-extras.md#updating-nodejs-from-upstream)
-    * make new branches for all commits you make!
+    * Make new branches for all commits you make!
```

Comment:
For me:

`request` sounds weird, you don't request a Pull Request, you request that your changes be landed by raising the Pull Request. It'd be like questioning a question.

I prefer `raise` to `create`, it has the feeling of bringing it to people's attention, and it matches raising issues. I'd say `raise`, `create`, or `open` are all fine though.